src/interactors/usersInteractor.js

```diff
@@ -0,0 +1,58 @@
+const interactor = require('./interactor');
+
+class UserInteractor extends interactor {
```

Comment:
This should be a basic interface and multiple implementations (such as `PostgresUserService`, `InMemoryUserService`, `FakeUserService`...).